### PR TITLE
[Merged by Bors] - chore: remove unused vars

### PR DESCRIPTION
**Pull request by adomani**
#17715

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Computability/Partrec.lean [hunk 1]
```diff
@@ -357,8 +357,8 @@ end Computable
 
 namespace Partrec
 
-variable {α : Type*} {β : Type*} {γ : Type*} {σ : Type*}
-variable [Primcodable α] [Primcodable β] [Primcodable γ] [Primcodable σ]
+variable {α : Type*} {β : Type*} {σ : Type*}
+variable [Primcodable α] [Primcodable β] [Primcodable σ]
```
Comment (grunweg) [inline Mathlib/Computability/Partrec.lean:361]: Feel free to combine these two lines.

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

File: Mathlib/Computability/Partrec.lean (hunk 1 above)
Comment (adomani) [inline Mathlib/Computability/Partrec.lean:361]:
```suggestion
variable {α : Type*} {β : Type*} {σ : Type*} [Primcodable α] [Primcodable β] [Primcodable σ]
```

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**